internal/runtime/hcsv2/workload_container.go

```diff
@@ -14,6 +16,8 @@ import (
 	"go.opencensus.io/trace"
 )
 
+const lcowNvidiaMountPath = "/nvidia"
```

Comment:
Should this be under `/run` like all other gcs state?

Reply:
I'm fine with that change